solidity/contracts/examples/ServiceAgreementConsumer.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.4.24;
+
+import "../Chainlinked.sol";
+
+contract ServiceAgreementConsumer is Chainlinked {
+  // NB: `Chainlinked` covers both the single-Oracle and Service-Agreement
+  // frameworks. Some of the terminology there is in terms of a single Oracle.
```

Comment:
I think `Chainlinked.sol` is a bit strained, because it's serving both the single-Oracle and the Coordinator cases. @dimroc and I decided to leave it as is, because the Coordinator case will have priority going forward, and the strain is manageable for now, but if we want to keep supporting both, this would be a good candidate for refactoring.